projects/packages/search/src/instant-search/components/post-type-icon.jsx

```diff
@@ -47,10 +48,22 @@ const PostTypeIcon = ( { postType, shortcodeTypes, iconSize = 18 } ) => {
 
 	switch ( postType ) {
 		case 'page':
-			return <Gridicon icon="pages" size={ iconSize } />;
+			return (
+				<Gridicon
+					icon="pages"
+					size={ iconSize }
+					description={ __( 'Page icon', 'jetpack-search-pkg' ) }
```

Comment:
Having looked at a few examples, I reckon we should omit "icon" here.
```suggestion
					description={ __( 'Page', 'jetpack-search-pkg' ) }
```

Here's one: 

https://www.unimelb.edu.au/accessibility/techniques/accessible-svgs

...the last example is for a home icon, labelled "Home".